code/modules/mob/living/carbon/xenomorph/mutators/strains/ravager/berserker.dm

```diff
@@ -29,7 +29,7 @@
 	R.health_modifier -= XENO_HEALTH_MOD_MED
 	R.armor_modifier += XENO_ARMOR_MOD_VERYSMALL
 	R.speed_modifier += XENO_SPEED_FASTMOD_TIER_3
-
+	R.received_phero_caps["frenzy"] = 2.9 // Moderate
```

Comment:
ideally you use the define for moderate frenzy here, instead of plain numbers

Reply:
theres no moderate frenzy define, the numbers are quite literally what defines moderate frenzy (or any other frenzy for that matter)
2.0 to 2.9 is moderate tier